src/getCoinCombination.test.js

```diff
@@ -7,4 +7,57 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return an array with 4 int`, () => {
+    const result = getCoinCombination(25);
+    expect(Array.isArray(result)).toBe(true);
+
+    for (const change of result) {
+      expect(Number.isInteger(change)).toBe(true);
+    }
+  });
+
+  it(`returns 0 with 0 cents`, () => {
+    expect(getCoinCombination(0)
+  ).toEqual([0, 0, 0, 0]);
+  });
+
+  it(`returns 1 penny with 1 cent`, () => {
+    expect(getCoinCombination(1)
+  ).toEqual([1, 0, 0, 0]);
+  });
+
+  it(`returns 1 nickel with 5 cents`, () => {
+    expect(getCoinCombination(5)
+  ).toEqual([0, 1, 0, 0]);
+  });
+
+  it(`returns 1 dime with 10 cents`, () => {
+    expect(getCoinCombination(10)
+  ).toEqual([0, 0, 1, 0]);
+  });
+
+  it(`returns 1 quarter with 25 cents`, () => {
+    expect(getCoinCombination(25)
```

Comment:
There is a missing closing parenthesis for the `expect` function call. Ensure that the parentheses are correctly balanced.